mono/native/pal_mount_tmp.c

```diff
@@ -64,6 +64,9 @@ static int32_t GetMountInfo(MountPointFound onFound)
     }
 
     return result;
+#else
+    return 0;
```

Comment:
does this mean this API will return empty result on Android?

Reply:
@akoeplinger as far as I understand, `setmntinfo` is not available in Android NDK

Reply:
It was added in Android-20 api level so I guess the `configure` check should handle it?..

Reply:
Our previous code did somehow handle it though so we need to add that back to corefx and not regress functionality.

Here's the output I get from `Directory.GetLogicalDrives()` on XA 9.2:

```
/system
/vendor
/mnt/runtime/default/emulated
/mnt/runtime/read/emulated
/mnt/runtime/write/emulated
/cache
/data
/data/var/run/netns
/storage/emulated
```

Reply:
Oh and I think we should put this code into corefx's `pal_mount.c` file instead of copy-pasting it into mono.

Reply:
That's why I put "_tmp_" in its name ;-) 
I guess it's better to use existing icall then, it's Android and AIX friendly.